content/02.introduction.md

```diff
@@ -187,6 +187,16 @@ Although these are difficult to disaggregate, they suggest different possible in
 The epidemiological characteristics and clinical presentation of COVID-19 may differ somewhat in different areas of the world, presumably due to differential reporting, different age structures, and/or different risk factors [@doi:10.1007/s00592-020-01546-0].
 Furthermore, because different subpopulations may have somewhat different vulnerabilities, needs, and resources, we recommend that researchers publishing studies on diagnostics and therapeutics take extra care to be clear about the demographic and medical characteristics of their sample, in order to facilitate discussions of the degree to which results may generalize or differ in other settings.
 
+Several studies on disparities in COVID-19 have compared groups in terms of their probability of a severe outcome (like hospitalization or death) given that a person is diagnosed.  
+The larger population of all exposed or all infected people usually cannot directly be studied, or even identified, because of the existence of undetected infections.  
+In contrast, [@doi:10.1038/s41586-020-2521-4] were a le to compare overall rates of death from COVID-19, not simply in the subsample of diagnosed individuals, thus combining the probability of infection with the probability of death given infection.  
+This was done using a dataset of 17 million adults in the United Kingdom.  
+In this administrative dataset, [@doi:10.1038/s41586-020-2521-4]  found that male sex, older age, economic deprivation, nonwhite ethnicity (principally Black or South Asian), obesity, diabetes,  asthma, and several other comorbid conditions were associated with a higher risk of death from COVID-19.  
+They cautioned that despite the large sample size, their observational data did not directly provide information on the causal mechanisms which led to these statistical associations.  
+However, the ethnic and economic disparities did not vanish when adjusting for observed preexisting conditions.  
+They also found some evidence that the risk associated with socioeconomic evidence may have increased with time, perhaps because many workers in jobs which cannot be done online also have lower-paying jobs.
```

Comment:
"evidence" doesn't seem like quite the right word -- variables, maybe? I'm not quite sure what would fit here.

Reply:
I totally botched that sentence.  How about "They found that the predictive value of socioeconomic status seemed to increase over time, perhaps because many higher-paid workers were able to transition to working online."